templates/assets/css/vlabs-style.css

```diff
@@ -171,13 +183,16 @@ iframe {
 }
 
 .visible {
-  width: max-content;
+  /* width: max-content; */
 }
 
 .hidden {
   width: 0px;
   border-width: 0px;
-  padding: 0px;
+  padding: 0px !important;
```

Comment:
I am generally not comfortable using !important. Would it work if instead of marking it hidden, we set the display:none attribute so it is taken out of the layout altogether?

Reply:
If I set display to none, then I won't be able to have closing animation for sidebar.

Reply:
ok then keep it the way it is

Reply:
Handled the expanded animation and removed padding, width, etc css for hiding menu.
Instead used display none (for desktop) and max height 0 (for mobile)